cli/azd/cmd/testdata/TestUsage-azd.snap

```diff
@@ -7,6 +7,7 @@ Usage
 Commands
   Configure and develop your app
     auth     	: Authenticate with Azure.
+    build    	: Builds the application's code. (Beta)
     config   	: Manage azd configurations (ex: default Azure subscription, location).
     hooks    	: Develop, test and run hooks for an application. (Beta)
```

Comment:
Not related, but why (Beta) on hooks?

Reply:
It was never promoted out of beta.  Happy to remove it when we're happy.